dev-packages/browser-integration-tests/suites/public-api/startSpan/standalone-mixed-transaction/test.ts

```diff
@@ -38,9 +38,16 @@ sentryTest(
     expect(traceId).toMatch(/[a-f0-9]{32}/);
     expect(parentSpanId).toMatch(/[a-f0-9]{16}/);
 
-    // TODO: the span envelope also needs to contain the `trace` header (follow-up PR)
     expect(spanEnvelopeHeader).toEqual({
       sent_at: expect.any(String),
+      trace: {
+        environment: 'production',
+        public_key: 'public',
+        sample_rate: '1',
+        sampled: 'true',
+        trace_id: traceId,
+        transaction: 'outer',
```

Comment:
So just to point this out: In a hyptothetical scenario, where a standalone span is created during an active transaction, it's DSC/`trace` header will be inherited from the active span tree's DSC. I think this is okay for now at least, since it guarantees DS and trace stability as long as one trace is active. Also this scenario is not going to happen in the SDK for how we plan on using `standalone` for `http.client` and INP spans.

Reply:
makes sense still for it to be this way! 👍